openapi.yaml

```
@@ -141,6 +179,22 @@ components:
       properties:
         asyncapi:
           $ref: '#/components/schemas/AsyncAPIDocument'
+    DiffDocument:
+      type: object
+      required:
+        - asyncapis
+      properties:
+        asyncapis:
+          type: array
+          minItems: 2
```

We should have only two files, so please add `maxItems: 2` as well :)

```suggestion
          type: array
          minItems: 2
          maxItems: 2
```